ec2/sc-ec2-windows-jumpcloud.yaml

```diff
@@ -0,0 +1,323 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Service Catalog: EC2 Reference Architecture(fdp-1oc5f3ula).'
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: General Configuration
+        Parameters:
+          - KeyPair
+      - Label:
+          default: Windows Instance Configuration
+        Parameters:
+          - WindowsInstanceType
+          - LatestAmiId
+    ParameterLabels:
+      KeyPair:
+        default: Key Pair
+      WindowsInstanceType:
+        default: Windows Instance Type
+      LatestAmiId:
+        default: SSM path for latest Windows AMI ImageId
+Parameters:
+  WindowsInstanceType:
+    AllowedValues:
+      - t3.nano
+      - t3.micro
+      - t3.small
+      - t3.medium
+      - t3.large
+      - t3.xlarge
+    Default: t3.micro
+    Description: Amazon EC2 Windows Instance Type
+    Type: String
+  KeyPair:
+    Description: Name of existing EC2 key pair for Windows Instances
+    Type: AWS::EC2::KeyPair::KeyName
+  LatestAmiId:
```

Comment:
might want to change this to just `AmiId` because it's not the latest AMI.

Reply:
Changed in 143401c